src/js/apps/patients/sidebar/action-sidebar_app.js

```diff
@@ -36,7 +36,6 @@ export default App.extend({
     this.showNewCommentForm();
   },
   showActivity() {
-    if (this.action.isNew()) return;
```

Comment:
Snuck this in just to clean up coverage reports. This is a redundant check.

Reply:
actually I think this whole app needs to bail if this is true.. currently it looks like we're showing a comment form on a new action.. can't post a comment to an action that has yet to exist... current bug